the brutlag bioinformatics group projects eblocks eblocks database of 80000 conserved protein regions from 20000 aligned protein families assembled using psi blast eblocks lets you compare new proteins to this database of conserved sequences using either emotifs or ematrices made from eblocks eproteome a database of emotifs ematrices and protein sequence similarities found in all of the open reading frames translated from over 60 genomes emotifs ematrices or sequence similarities can be selected by genome keyword or specificity or combinations of these parameters 3motif 3motif visualizes the three dimensional structure of emotifs contained within structures from the protein structure database pdb one needs the chime plug in to view the 3motif structures multiple views of the structural motifs are possible 3matrix visualizes the three dimensional structure of ematrix motifs contained within structures from the protein structure database pdb one needs the chime plug in to view the 3matrix motif structures multiple views of the structural motifs are possible bioprospector mdscan bioprospector takes sets of upstream sequences of co regulated genes from either microarray experiments or clustered microarray experiments and can discover conserved transcription factor binding sites in these upstream sequences bioprospector uses a stochastic gibb's sampling alignment algorithm mdscan takes upstream sequences from either single microarray experiments or from chromatin immune precipitation experiments and can discover weakly conserved transcription factor binding sites mdscan has nearly the same sensitivity as bioprospector but uses a deterministic algorithm which is several fold faster and does not get caught in local minima lock 2 a vector based protein structure superposition algorithm capable of recognizing distant structural similarities statistical significance values for alignments are provided foldminer performs structural similarity searches and discovers a structural motif common to the query protein and high scoring target structures motifs are used to improve both the sensitivity and specificity of the search by focusing on conserved portions of the structure emotif search compares proteins or orfs of unknown functions against a database of sequence motifs 50000 of varying specificities to identify functional and structural motifs in the protein ematrix search compares proteins or orfs of unknown function against a series of position specific scoring matrices compiled from the blocks and prints databases using a very rapid and highly discriminating search procedure emotif scan searches the genpept and swiss prot databases for proteins containing a particular motif or regular expression ematrix scan ematrix scan takes a position specific scoring matrix generated by ematrix maker and scans the swiss prot database for examples of sequences that match the scoring matrix with an expectation above a specified level default is 10 8 emotif maker forms protein sequence motifs for subsets of aligned sequences ranking them by both their specificity and the number of supplied sequences covered sensitivity ematrix maker develops position specific scoring matrices pssm's for protein motifs that permits sequence weighting prior probabilities or pseudocounts and a variety of discriminant functions to be used in database search esignal esignal a database of emotifs and ematrices generated exclusively from signal transduction proteins esignal can be used to discover signal transduction functions in new proteins decypher decypher is a high performance sequence database search engine from time logic that can perform blast searches blastn blastp blastx tblastn and psi blast and dynamic programming searches smith waterman profile hmm in seconds on entire databases and entire genomes disclaimer for use of these web resources please send comments and questions to
