compare prospector home overview inputs output search reference download supplementary tables bioprospector search about the authors we developed compareprospector to take advantage of comparative genomics information to aid sequence motif finding compareprospector is built upon bioprospector liu x et al 2000 which is an extension of the original gibbs sampler liu js et al 1995 with improved flexibility and performance compareprospector takes as input a list of sequences from one species that is predicted to share common regulatory element s such sequences can be obtained from high throughput genomics techniques such as gene expression profile clustering or chromatin immunoprecipitation followed by microarray chip chip it also takes as input a list of percent identity values representing the cross species conservation of each nucleotide in the gibbs sampling iterations compareprospector biases the motif finding towards sequences conserved across species first of all the user can specify two wpid thresholds t ch high conservation threshold and t cl low conservation threshold in bioprospector a site score ax is calculated for every site x in the input sequence as the ratio of the probability of generating x from the motif model over the probability of generating x from the background distribution a new site is sampled with probability proportional to ax in compareprospector during initial iterations of gibbs sampling only positions whose wpid values are above t ch are sampled subsequently the wpid cutoff is gradually decreased from t ch to t cl to allow sampling of less conserved positions the new site score a x is weighted by sequence conservation a x ax wpidx wpidx being the wpid of site x to favor sampling of more conserved sequences sequences without orthologs are assigned t cl as the wpidx for all x so they only participate in sampling in later iterations finally in the original bioprospector sites with a high enough score ax are automatically added to the motif without sampling compareprospector restricts automatic additions to only sites whose wpids are above t ch this step further down weighs the influence of divergent sites and sequences without orthologs the output of compareprospector includes a list of highest scoring motifs as position specific probability matrices the individual sites used to construct each motif and the locations of the sites on the input sequences liu js neuwald af and lawrence ce 1995 bayesian models for multiple local sequence alignment and gibbs sampling strategies journal of the american statistical association 90 1156 1170 liu x brutlag dl and liu js 2001 bioprospector discovering conserved dna motifs in upstream regulatory regions of co expressed genes pac symp biocomput 127 138
