yeastdeletionwebpages deletion page home databases and datasets faqs deletion strains available references protocols &technical information useful sites & links consortium members addresses yeast deletion database consortium members only confirmation pcr primer design the confirmation primers were designed using a modified version of the wi institute primer program version 2.3 for information and references about this program see the whitehead institute genome software homepage the primers were designed to have lengths between 17 and 28 bases and to have melting temperatures of 65 c plus or minus 2 c the minimum gc was set to be 30 and maximum to be 70 primers a and d were picked from regions 200 to 400 bases upstream or downstream respectively of the open reading frame primers b and c were chosen from regions within the open reading frame and were designed to give pcr products with sizes of 300 1000 bases when used with a or d respectively when the primers were chosen a primer record was created this file contains the parameters that were used to select the primers and a and b or c and d primer sequence for each orf it also contains the expected pcr product sizes that would be observed for an ab deletion or a kanb deletion amplification event we learned that primer 2.3 which had been used to pick confirmation primers for chromosomes v xiii and i has a bug in it the program picked primers that have melting temps that are lower than the requested melting temperature 65 c in reality the primers produced had melting temps that are between 53 c and 58 c the primers still worked extremely well we estimated the confirmation pcr is between 90 and 95 percent successful when performed on wildtype template however we did not determine whether unsuccessful pcr's occured because of primer synthesis failure template quality or primer design failure last updated august 2002 amchu cmgm stanford edu
