splicemap splice junction discovery and alignment using rna seq splicemap is developed by kin fai au and john c mu with help from collaborators at wong lab splicemap latest news splicemap 3.3 5.2 faster and more accurate read more viewing output there are two choices for viewing the output from splicemap ucsc genome browser or cisgenome browser these instructions will be expanded on in the future please contact us if you have any concerns ucsc genome browser viewing the output in ucsc genome browser is simple the output is already formatted so that it can be readily accepted by the browser however due to the large size of coverage files you may have to host them locally with cisgenome browser cisgenome browser cisgenome browser is a light weight alternative to ucsc browser it is run on your own personal computer the junction bed files are directly readable by cisgenome browser the coverage wig files need to be converted to bar with the following steps for instance if you wanted to convert the file coverage wig run wig2bigwig coverage wig coverage barwig run barloader coverage barwig the resulting file coverage barwig bar can be read by cisgenome browser barloader has a high memory requirement for 36 million reads on the mouse genome you will need over 20g of ram the following is some example output from cisgenome browser site map home features how it works downloads contact help tutorial manual output format filters cfg file format viewing output faq download current release splicemap 3.3 5.2 notes previous very old release splicemap 2.1 1 notes useful downloads all gene annotations hg19 all gene annotations hg18 useful tools seqmap maps short sequences to the genome rseq isoform specific gene expression estimation cisgenome browser light weight genome browser similar to ucsc browser kin fai au john c mu all rights reserved stanford ca 94305
