splicemap splice junction discovery and alignment using rna seq splicemap is developed by kin fai au and john c mu with help from collaborators at wong lab splicemap latest news splicemap 3.3 5.2 faster and more accurate read more faq if you have any questions that have not been answered here please contact us what is splicemap splicemap is a program written entirely in c++ that is able to discover splice junctions from rna seq data it also computes the coverage at each location and functions as an aligner does splicemap support fastq format yes as of version 3.2 it does however it does not yet preserve read quality this will be added in the next version does splicemap output in sam format yes as of version 3.1 1 it does does splicemap support reads in colorspace no not yet if there is high demand we will look into it what is eland and how do i obtain it eland is a short reads mapper that is provided if you are a user of illumina tm devices as a result we cannot include this tool in the package however we do provide some instructions on how to configure it if you do have the eland software please contact us if you are licensed to use eland and we can provide you a pre compiled copy what is bowtie and how do i obtain it bowtie can be found here site map home features how it works downloads contact help tutorial manual output format filters cfg file format viewing output faq download current release splicemap 3.3 5.2 notes previous very old release splicemap 2.1 1 notes useful downloads all gene annotations hg19 all gene annotations hg18 useful tools seqmap maps short sequences to the genome rseq isoform specific gene expression estimation cisgenome browser light weight genome browser similar to ucsc browser kin fai au john c mu all rights reserved stanford ca 94305
