splicemap splice junction discovery and alignment using rna seq splicemap is developed by kin fai au and john c mu with help from collaborators at wong lab splicemap latest news splicemap 3.3 5.2 faster and more accurate read more how does splicemap work splicemap is a pure implementation of the split reads alignment algorithm 25bp half reads are mapped to the genome with with either bowtie eland or seqmap then the half reads are pieced together to determine locations of exons and possible junctions unmapped half reads are used to search for splice junctions currently only the cannoical gt ag splice sites are identified in the case of reads longer than 50bp the reads it separated into overlapping segments of 50bp and the same algorithm above is run finally all the segments are stitched together to find the final alignment splicemap is geared towards mammalian genomes if you are having trouble running splicemap on plant yeast or bacterial genomes please let us know site map home features how it works downloads contact help tutorial manual output format filters cfg file format viewing output faq download current release splicemap 3.3 5.2 notes previous very old release splicemap 2.1 1 notes useful downloads all gene annotations hg19 all gene annotations hg18 useful tools seqmap maps short sequences to the genome rseq isoform specific gene expression estimation cisgenome browser light weight genome browser similar to ucsc browser kin fai au john c mu all rights reserved stanford ca 94305
