rosenberg lab software rosenberg lab at stanford university home about research publications software adze clumpp distruct haploconfig infocalc data sets people map and directions news joining the lab links noah a rosenberg 1 650 721 2599 office phone 1 650 723 6132 fax office 122d herrin labs mailing address department of biology stanford university 371 serra mall stanford ca 94305 5020 usa last modified 7 13 2011 software adze adze is a program that implements the rarefaction method for analyzing allelic diversity across populations while correcting for sample size differences using individual multilocus genotype data on genetic polymorphisms adze computes estimates of allelic richness private allelic richness and private allelic richness for combinations of populations adze was used for generating figure 1a in genotype haplotype and copy number variation in worldwide human populations nature 451 998 1003 2008 a program note describing adze was published in bioinformatics 24 2498 2504 2008 clumpp clumpp is a program that deals with label switching and multimodality problems in population genetic cluster analyses clumpp permutes the clusters output by independent runs of clustering programs such as structure so that they match up as closely as possible the user has the option of choosing one of three algorithms for aligning replicates with a tradeoff of speed and similarity to the optimal alignment a program note describing clumpp was published in bioinformatics 23 1801 1806 2007 distruct distruct is a program that can be used to graphically display results produced by the clustering program structure or by other similar programs the figures produced by distruct display individual membership coefficients in the same form as used in genetic structure of human populations science 298 2381 2385 2002 various options enable the user to control left to right printing order of populations bottom to top printing order of clusers colors and other graphical details a program note describing distruct was published in molecular ecology notes 4 137 138 2004 haploconfig haploconfig is a program that can be used to implement tests of neutrality based on the frequency distribution of haplotypes in a sample of dna sequences the haplotype configuration and the number of segregating sites the neutrality tests can be performed conditional on the standard neutral coalescent model with or without recombination exponential population growth or island migration a description of the method underlying the program can be found in statistical tests of the coalescent model based on the haplotype frequency distribution and the number of segregating sites genetics 169 1763 1777 2005 the haploconfig program can also be used as a coalescent simulator for models with or without recombination infocalc infocalc is a small script for calculating statistics that measure the ancestry information content of genetic markers a description of these statistics can be found in informativeness of genetic markers for inference of ancestry american journal of human genetics 73 1402 1422 2003 with extensions in algorithms for selecting informative marker panels for population assignment journal of computational biology 12 1183 1201 2005
