splicemap splice junction discovery and alignment using rna seq splicemap is developed by kin fai au and john c mu with help from collaborators at wong lab splicemap latest news splicemap 3.3 5.2 faster and more accurate read more features splicemap aligns rna seq reads to a reference genome in order to discover splicing junctions it is at least as sensitive and specific as state of the art tools some major features of splicemap are support for arbitrarily long uneven read lengths optional filtering reports the reliability of each junction reports non uniquely mapped coverage outputs in sam format for easy down stream analysis high speed better performance on sensitivity specificity and running time compared to existing tools refer to the paper for details however the current splicemap is newer than the one in the paper includes developers who provide prompt responses to any of your questions outputs md string in sam file for snp calling without a reference the following table gives an idea of the improvements expected from long reads the results are from the same 20 million 100bp rna seq reads cut to different lengths and mapped to the human genome hg18 specificity is calculated based on the number of junctions validated by ests the results are unfiltered improvements to specificity can be achieved using the optional filters 50bp 75bp 100bp total junctions 158060 161295 170213 est validated 136054 137964 142432 specificity 86.08 85.54 83.68 site map home features how it works downloads contact help tutorial manual output format filters cfg file format viewing output faq download current release splicemap 3.3 5.2 notes previous very old release splicemap 2.1 1 notes useful downloads all gene annotations hg19 all gene annotations hg18 useful tools seqmap maps short sequences to the genome rseq isoform specific gene expression estimation cisgenome browser light weight genome browser similar to ucsc browser kin fai au john c mu all rights reserved stanford ca 94305
