splicemap splice junction discovery and alignment using rna seq splicemap is developed by kin fai au and john c mu with help from collaborators at wong lab splicemap latest news splicemap 3.3 5.2 faster and more accurate read more requirements splicemap aims to fully integrate into a variety of analysis pipelines if any of the following requirements conflict with your lab's set up please contact us system requirements linux or mac osx operating system 16gb ram recommended 8gb minimum for human genome 30gb free hard drive space less is probably ok but just to be safe read format paired or non paired reads supported read length greater or equal to 50bp there is no requirement that reads must be the same length however you should not mix paired and single end reads upstream input to splicemap are sequencer reads in raw fastq or fasta format short read mappers supported bowtie preferred seqmap eland downstream if sam format output is chosen cufflinks may be used for further analysis rseq is another choice although gene annotations are required site map home features how it works downloads contact help tutorial manual output format filters cfg file format viewing output faq download current release splicemap 3.3 5.2 notes previous very old release splicemap 2.1 1 notes useful downloads all gene annotations hg19 all gene annotations hg18 useful tools seqmap maps short sequences to the genome rseq isoform specific gene expression estimation cisgenome browser light weight genome browser similar to ucsc browser kin fai au john c mu all rights reserved stanford ca 94305
