great input genomic regions enrichment of annotations tool bejerano lab stanford university overview demo video how to cite download help forum bejerano lab stanford university version 1.8 2 genomic regions enrichment of annotations tool great version 1.8 2 current 11 19 2010 to now great version 1.2 6 reproduce results from 05 20 2010 to 11 18 2010 great predicts functions of cis regulatory regions many coding genes are well annotated with their biological functions non coding regions typically lack such annotation great assigns biological meaning to a set of non coding genomic regions by analyzing the annotations of the nearby genes thus it is particularly useful in studying cis functions of sets of non coding genomic regions cis regulatory regions can be identified via both experimental methods eg chromatin immunoprecipitation followed by massively parallel sequencing and by computational methods eg sequence conservation in a given clade news feb 21 2012 great adds forums for better user discussion may 31 2011 great version 1.8 adds genome distribution graphs and upgrades term details pages mar 29 2011 great version 1.7 adds support for human disease ontologies jan 27 2011 great version 1.6 adds support for the zebrafish genome jan 10 2011 great included in nature biotechnology trends in computational biology 2010 dec 15 2010 great receives its first nature citation may 10 2010 you can now submit to great directly from the ucsc table browser may 2 2010 great is published in nature biotechnology see journal reprint faculty of 1000 must read to see announcements and news about great join the low traffic 1 email per month great announce mailing list email address your name optional input choices species assembly human grch37 ucsc hg19 feb 2009 human ncbi build 36.1 ucsc hg18 mar 2006 mouse ncbi build 37 ucsc mm9 jul 2007 zebrafish wellcome trust zv9 danrer7 apr 2010 can i use a different species or assembly test regions bed file bed data what should my test regions file contain how can i create a test set from a ucsc genome browser annotation track background regions whole genome bed file bed data when should i use a background set what should my background regions file contain association rule settings show settings associating genomic regions with genes great calculates statistics by associating genomic regions with nearby genes and applying the gene annotations to the regions association is a two step process first every gene is assigned a regulatory domain then each genomic region is associated with all genes whose regulatory domain it overlaps note that all distances are measured from the transcription start site of the gene's canonical isoform basal plus extension proximal kb upstream kb downstream plus distal up to kb gene regulatory domain definition each gene is assigned a basal regulatory domain of a minimum distance upstream and downstream of the tss regardless of other nearby genes the gene regulatory domain is extended in both directions to the nearest gene's basal domain but no more than the maximum extension in one direction two nearest genes within kb gene regulatory domain definition each gene is assigned a regulatory domain that extends in both directions to the nearest gene's tss but no more than the maximum extension in one direction single nearest gene within kb gene regulatory domain definition each gene is assigned a regulatory domain that extends in both directions to the midpoint between the gene's tss and the nearest gene's tss but no more than the maximum extension in one direction include curated regulatory domains what are curated regulatory domains loading copyright 2010 the board of trustees of leland stanford junior university all rights reserved terms of use about us contact us views counter
