hiv drug resistance database stanford university hiv drug resistance database a curated public database designed to represent store and analyze the divergent forms of data underlying hiv drug resistance home genotype rx genotype pheno genotype clinical hiv db program resources isolates a panel of 12 prototypical infectious multidrug resistant hiv 1 reverse transcriptase rt clones the panel includes clones with each of the published nucleoside analog rt inhibitor nrti mutations in the combinations that occur most frequently in hiv infected individuals multi drug resistant hiv 1 rt panel programs syn scan calculates the synonymous and nonsynonmous mutation rates between sequences containing ambiguous nucleotides syn scan page pyromap accurately maps pyrosequencing reads onto reference sequences using a selectively weighted smith waterman sw 2 algorithm to incorporate quality scores into alignment download pyromap tar gz g to a hypermutation filter generation using expectation maximization given a table of aggregated values and a set of preliminary filter constants this program will construct a set of frequency distributions characterizing the hypermutated and non hypermutated sequence subsets the filter constants can then be exported as a perl library python file or in human readable statements manuscript submitted download g2a_filter_em tar bz2 database citing database terms of use faqs user guide & database documents database statistics news resources hiv treatment websites rt protease and integrase structures additional resources team who we are & how to contact us publications acknowledgements 1998 2012 all rights reserved questions contact hivdb
