qc check documentation quality control script v2 2 12 04 2007 the purpose of this script is to automatically perform a set of tasks that are done routinely during the qc process the quality control check was developed at the joint center for structural genomics if you use this tool in preparing a structure please acknowledge the jcsg 1 usage the interface allows the input of necessary files and allows the user to select which checks to run pdb_file mtz_file and refmac_log are the most recent files from the refinement sequence is the sequence file in fasta format it is only necessary to upload the files that correspond to the selected job requests for example selecting molprobity requires only the pdb file while the sequence check requires the pdb and sequence files most browsers will list the required input files if the mouse is held over the check boxes 2 output all results are output to the screen use the links at the top of the page to jump to a specific section saving results under the input comments and save results section located at the bottom of the page there is a box in which the user can optionally add comments the entire output can then be saved using the save comments and qc output box it will automatically be saved as qc_check html though the name can be changed it is important however to retain the html to facilitate later viewing in a web browser note that the file will be saved even if no comments are entered 3 checks refinement statistics the space group and unit cell parameters are presented the resolution limits are checked and the nominal resolution is calculated the rwork and rfree stats are presented and the gap between them is calculated flagged if the gap is over 5 expectation values calculated based on tickle 1998 2000 flagged if the standard deviation from the expectation value is greater than three if the fo fc map correlation is below 0.9 it is flagged if the rms bond length is under 0.014 or over 0.021 it is flagged if the chiral center is under 0.085 or over 0.115 it is flagged the phase weights are checked the b factor restraints are checked check for the addition of hydrogens during refinement the number of ncs groups is compared to the number of molecules in the asu if the resolution is worse than 2.0 and there are no ncs restraints then it is flagged the number of tls groups is presented the presence of make_u_positive errors is flagged pdb checks occupancy is checked any atoms that do not have occ 1.0 are listed mse occupancy is listed seperately from the other atoms and the file is checked for the presence of met high and low b factors are listed highlighted if any b factors are equal to 2.00 check chain id's specifically look for the presence of chain id x which is typically inserted by refmac verify that remarks 300 350 and 999 are present does not check that content is correct sequence alignment sequence alignment is presented and any gaps and or mismatches are highlighted ncs outliers possible residue flips between different chains are flagged residues that are in different conformations between different chains are flagged an aligned pdb file and a coot script that lists and residues flagged from 1 or 2 above are generated molprobity checks check for remark 42 which is generated using molprobity if remark 42 is absent it is automatically generated the clash rotamer and ramachandran scores from molprobity are presented if present ramachandran outliers are listed if present rotamer outliers are listed if present flips to check are listed a script compatible with coot is available for download open in coot with calculate run script adit check adit is run and results from pdb_file letter are included whatif check whatif is run to check for errors with the nomenclature resolve evaluate resolve_evaluate is run to check fit of atoms in electron density and residues with weak density are listed
